samples/fr/archibald_haddock.json

```
@@ -1,6 +1,7 @@
 {
   "accounts": [
     {
+      "name": "Compte Courant 01",
```

Most checking accounts have the owner name in the name, haven't they?
```suggestion
      "name": "Compte Courant MR Archibald Haddock",
```

---

Not necessarily, and Product Team asked for "Compte Courant N".